dirty_cat/tests/test_docstrings.py

```diff
@@ -77,8 +74,16 @@ def get_all_methods():
             yield Estimator, method
 
 
+def get_functions_to_validate():
```

Comment:
What's the difference between this function and `get_public_functions` ?

Reply:
It's an intermediary function to `get_public_functions` used by by the parametrization, line 194, just like `get_methods_to_validate` is for `get_public_classes`.
I'm just keeping the process similar for both public functions and class methods.